app/models/solidus_subscriptions/subscription.rb

```diff
@@ -311,6 +312,10 @@ def set_payment_method
       end
     end
 
+    def set_currency
+      self.currency = ::Spree::Config[:currency] if self[:currency].nil?
```

Comment:
Can this be simplified as `self.currency ||= ::Spree::Config.currency`?

Reply:
Thanks 🙏  fixed.